web/src/components/navigation/Navigation.vue

```diff
@@ -230,8 +229,9 @@ export default {
     }
   },
   watch: {
-    name() {
-      this.handleUserInitialsFetch();
+    name: {
+      handler: 'handleUserInitialsFetch',
```

Comment:
hmh tätä tapaa tiennytkään, cool

Reply:
Kattelin apista niin tolla pitäis olla mitään merkitystä kummalla tapaa tekee, kumpikin invokettaa samantien.

Reply:
vähemmän koodia!